modules/local/virsorter2/main.nf

```diff
@@ -0,0 +1,38 @@
+process VIRSORTER2 {
+    tag "$meta.id"
+    label 'process_medium'
+    conda "bioconda::virsorter=2.2.3-pyhdfd78af_1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'docker://jiarong/virsorter:2.2.3' :
+        'jiarong/virsorter:2.2.3' }"
+
+    input:
+    tuple val(meta), path(fasta)
```

Comment:
take db from virsorter/setup module

Reply:
resolved in https://github.com/cidgoh/nf-seqqc/commit/30067fb3b5f244ca93b80296dce93214cc55c1b6